### PR TITLE
Header styling done

### PR DESCRIPTION
**Pull request by maliaaska**

### CODE REVIEW

File: src/pages/contacts/components/contact/contact.style.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const ContactWrapperStyled = styled.div`
   width: 100%;
   padding: 5px;
   border-bottom: 1px gray solid;
-  background-color: azure;
+  background-color: white;
```
Comment (fookoo) [inline src/pages/contacts/components/contact/contact.style.ts:9]: What is for ?

File: src/pages/contacts/components/header/header.component.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { faUserPlus } from '@fortawesome/free-solid-svg-icons'
+import { HeaderStyled } from './header.style'
```
Comment (fookoo) [inline src/pages/contacts/components/header/header.component.tsx:3]: fix import order (styles should be last)

File: src/pages/contacts/contacts.page.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react'
 import { ContactsService, IContact } from './services/contact.service'
 import { ContactList } from './components/contact-list/contact-list.component'
 import { ContactPageStyled } from './contacts.style'
+import { Header } from './components/header/header.component'
```
Comment (fookoo) [inline src/pages/contacts/contacts.page.tsx:5]: import order

File: yarn.lock [hunk 4]
```diff
@@ -1170,6 +1170,32 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@fortawesome/fontawesome-common-types@6.1.1":
```
Comment (fookoo) [inline yarn.lock:1173]: @fortawesome/fontawesome-common-types is missing in package.json

re run yarn install --force

**Review by fookoo — CHANGES_REQUESTED** [review]

File: src/pages/contacts/components/header/header.component.tsx (hunk 2 above)
Comment (maliaaska) [reply src/pages/contacts/components/header/header.component.tsx:3]: Done

File: src/pages/contacts/contacts.page.tsx (hunk 3 above)
Comment (maliaaska) [reply src/pages/contacts/contacts.page.tsx:5]: order done

**Review by fookoo — APPROVED** [review]

**Pull request merged**